assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/useSubmitButtonProps.tsx

```diff
@@ -5,9 +5,11 @@ import { useOnSubmitPrices } from '../hooks';
 
 type Props = {
 	onCloseModal: VoidFunction;
+	price: number;
 };
 
-const useSubmitButtonProps = ({ onCloseModal }: Props): ButtonProps => {
+const useSubmitButtonProps = ({ onCloseModal, price }: Props): ButtonProps => {
+	const disabled = price === 0;
```

Comment:
A Ticket can be FREE 😄

Reply:
this check was added in context of this requirement:
```
disable submit / prevent modal closing and add error message if price modifier is added without a value
```
I tend to think that `isFree` reflects a value

Reply:
A ticket can be free, meaning its base price is $0 and it has no price modifiers

but the price modifiers (surcharges, discounts, taxes) can not have an amount of 0